server/src/data/models.rs

```diff
@@ -989,6 +989,7 @@ pub struct Event {
     pub event_data: serde_json::Value,
 }
 
+//TODO: change these to Chunk instead of cards but need to rescroll pg to fix ids in events
```

Comment:
Would a migration script not be enough for this?

Reply:
Yes need to write a migration script to scroll PG